tardis/io/config_reader.py

```diff
@@ -1018,7 +1018,7 @@ def from_config_dict(cls, config_dict, atom_data=None, test_parser=False,
 
 
 
-        if montecarlo_section['convergence_strategy'] is None:
+        if 'convergence_stragegy' not in montecarlo_section:
             logger.warning('No convergence criteria selected - '
                            'just damping by 0.5 for w, t_rad and t_inner')
             montecarlo_section['convergence_strategy'] = (
```

Comment:
Since we decided to use default values for now, I propose to move these default values to either the schema or the Simulation class